admin/includes/functions/general.php

```diff
@@ -3847,3 +3847,7 @@ function zen_get_uploaded_file($filename) {
     $filetype = $file_parts[sizeof($file_parts) - 1]; 
     return $filenum . "." . $filetype; 
   }
+function issetorArray(array $array, $key, $default = null)
+{
+    return isset($array[$key]) ? $array[$key] : $default;
+}
```

Comment:
This function exists in `DIR_FS_CATALOG . 'includes/functions/functions_general.php'`
Maybe we can just move it over to `functions_general_shared.php` since the Admin loads that file already?

Reply:
And update the test `not_for_release/testFramework/unittests/testsSundry/IssetorArrayTest.php`

Reply:
I know this will create a merge conflict, but moving this to the shared file also makes great sense for the test suite updates in PR #3524